CorsixTH/Lua/world.lua

```diff
@@ -954,6 +954,14 @@ function World:pauseOrUnpause()
   end
 end
 
+--! Function to check if player can perform actions when paused
+--!return (bool) Returns true if player hasn't allowed editing while paused
+function World:checkPauseBehaviour()
+  if self:isCurrentSpeed("Pause") and not self.user_actions_allowed then
+    return true
+  end
+end
```

Comment:
Ever noticed how easy it is to understand thre result value of `self:isCurrentSpeed("Pause")`?

The `and` expression already results in a boolean true/false, so you can reduce this function to `return self:isCurrentSpeed("Pause") and not self.user_actions_allowed`.
That also solves no returning `nil` as boolean value.

Function name seems not quite right to me, much alike the previous comment.